assemblies/assembly-configuring-techdocs.adoc

```diff
@@ -1,8 +1,10 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="assembly-techdocs-plugin_{context}"]
-= Configuring the TechDocs plugin in {product}
+:context: configuring-techdocs
+[id="{context}"]
+= Configuring TechDocs
 
-The {product} TechDocs plugin helps your organization create, find, and use documentation in a central location and in a standardized way. For example:
+Configure the {product} TechDocs plugin to create, find,
```

Comment:
@themr0c , is it necessary for this sentence to be split across line 6 and 7, or can they be in one line?

Reply:
No, mistake, my editor is doing that and I need to reconfigure it properly.